webapp/packages/plugin-help/src/Shortcuts/Shortcut.tsx

```diff
@@ -16,6 +16,12 @@ interface Props {
   shortcut: IShortcut;
 }
 
+const CODE_FORMAT_MAP: Record<string, string> = {
+  comma: ',',
+  slash: '/',
+  backslash: '\\',
+};
```

Comment:
This logic should be handled in `getCommonAndOSSpecificKeys` because it's library-specific (why do we need to change our application every time the library updates)